packages/react/src/themes/teams/components/Icon/svg/icons/index.ts

```diff
@@ -118,6 +119,7 @@ export default {
   'participant-remove': participantRemove,
   'phone-clock': phoneClock,
   quote,
+  'read-aloud': readAloud,
```

Comment:
is `read-aloud` icon exported because it's needed or just for this example?

Reply:
ah, true. i'm not sure this feature is supported yet.